src/components/CalendarPicker/index.js

```diff
@@ -18,8 +18,8 @@ class CalendarPicker extends React.PureComponent {
     constructor(props) {
         super(props);
 
-        this.monthNames = moment.localeData(props.preferredLocale).months();
-        this.daysOfWeek = moment.localeData(props.preferredLocale).weekdays();
+        this.monthNames = _.map(moment.localeData(props.preferredLocale).months(), month => month.replace(/^\w/, c => c.toUpperCase()));
```

Comment:
NAB: Just wondering why we didn't do something simpler like:

```suggestion
        this.monthNames = _.map(moment.localeData(props.preferredLocale).months(), month => month.toUpperCase());
```

Reply:
@aldo-expensify  The first line of code converts the month names to title case (where the first letter of each word is capitalized), while the second line of code converts the month names to all uppercase. We just need to convert first letter of the month to uppercase

Reply:
I see, thanks for explaining!

Reply:
Welcome

Reply:
I was also confused by this. Could you please add a small comment explaining this for people in the future? Thanks

Reply:
@thesahindia ok I'm going to change it